package.json

```diff
@@ -112,7 +111,6 @@
     "lint-staged": "^11.1.2",
     "prettier": "2.2.1",
     "regenerator-runtime": "^0.13.9",
-    "serverless": "^3.22.0",
     "serverless-offline": "^11.0.1",
     "serverless-offline-lambda": "^1.0.6",
     "serverless-plugin-typescript": "^2.1.2",
```

Comment:
@kishore03109 double checking - all the `serverless-xxx` stuff can be removed?

Reply:
yup, can be removed!